h2o-py/tests/testdir_apis/h2o_cluster_info/pyunit_h2oinit.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import print_function
+from builtins import str
+from builtins import range
+import sys
+sys.path.insert(1,"../../../")
+from tests import pyunit_utils
+import h2o
+
+
+def h2oinit():
```

Comment:
Does this test just check if `h2o.init()` compiled? I think we would rather want to test if name parameters are doing what we expect.